src/main/common/maths.c

```diff
@@ -49,14 +49,21 @@
 #define sinPolyCoef7 -1.980661520e-4f                                          // Double: -1.980661520135080504411629636078917643846e-4
 #define sinPolyCoef9  2.600054768e-6f                                          // Double:  2.600054767890361277123254766503271638682e-6
 #endif
+
 float sin_approx(float x)
```

Comment:
```suggestion
FAST_CODE_NOINLINE float sin_approx(float x)
```

Reply:
@haslinghuis `FAST_CODE NOINLINE` or `FAST_CODE FAST_CODE_NOINLINE` is probably even better - we want fast sin/cos, but only one copy of it

Reply:
This is needed to implement your earlier suggestion to fit F745 (101.66% overflow)

- `FAST_CODE_NOINLINE` build is 97.24%
- `FAST_CODE NOINLINE` or `FAST_CODE FAST_CODE_NOINLINE` build is 98.29%

Reply:
With the change in this [commit](https://github.com/betaflight/betaflight/pull/13975/commits/3f823c7c6b5548eae6aebce0b6eb9f633a48337e)